var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -30,37 +29,19 @@ class Zfp(MakefilePackage):
     variant('shared', default=True,
             description='Build shared versions of the library')
 
-    def edit(self, spec, prefix):
-        config_file = FileFilter('Config')
-        config_file.filter(
-            r'^\s*#\s*DEFS\s*\+=\s*-DBIT_STREAM_WORD_TYPE\s*=\s*uint8',
-            'DEFS += -DBIT_STREAM_WORD_TYPE=uint%s' %
-            spec.variants['bsws'].value)
+    depends_on('cmake@3.4.0:', type='build')
```

Comment:
Do we need a specific cmake version here? Isn't `depends_on('cmake', type='build') sufficient?

Reply:
`depends_on('cmake', type='build')` isn't as specific. If someone has an older version of CMake installed on their OS and added as an external package, Spack should fail and say a newer version of CMake is required.

Reply:
Yes, the `CMakeLists.txt` of ZFP explicitly requires 0.3.1+ for Unix and 0.3.4+ for Windows based installs. I just took the greater of the two, even if we do not yet support windows, since it's an ancient version of CMake anyway.

Reply:
Ah, ok. Thanks for info.